tests/sasl_test.py

```diff
@@ -211,6 +221,11 @@ def handle_report(report, version, suite):
                'rdkconf': {'sasl_oauthbearer_config': 'scope=wrongScope'},
                'tests': ['0001'],
                'expect_fail': ['all']},
+              {'name': 'OAuth/OIDC',
+               'run': args.oidc,
+               'tests': ['0126'],
+               'conf': sasl_oauth_oidc_conf,
+               'expect_fail': ['2.1.0', '0.10.2.0', '0.9.0.1', '0.8.2.2']},
```

Comment:
I recently added 2.8.1 to the list, which I think does not have OIDC, so add it here too.